src/test/java/org/broadinstitute/consent/http/resources/StudyResourceTest.java

```diff
@@ -222,11 +225,12 @@ void testDeleteStudyById() throws Exception {
     admin.setAdminRole();
     admin.setUserId(study.getCreateUserId());
     when(userService.findUserByEmail(any())).thenReturn(admin);
+    when(elasticSearchService.deleteIndex(any(), any())).thenReturn(response);
     initResource();
 
     try (Response response = resource.deleteStudyById(authUser, study.getStudyId())) {
       assertEquals(HttpStatusCodes.STATUS_CODE_OK, response.getStatus());
-      verify(elasticSearchService, atLeastOnce()).deleteIndex(any());
+      verify(elasticSearchService, atLeastOnce()).deleteIndex(any(), any());
```

Comment:
Why do you use `atLeastOnce()` here? Do we expect it to be called more than once?
```suggestion
      verify(elasticSearchService).deleteIndex(any(), any());
```

Also in this case since you use strict stubbing this `verify()` is redundant.

Reply:
I think `atLeastOnce()` is the wrong test here. We should be ensuring that we're only calling the delete once and only once.